src/accessories/TokenArranger.ttslua

```diff
@@ -315,66 +320,68 @@ function layout(_, _, isRightClick)
   end
 
   -- sort table by value (symbols last if same value)
-  table.sort(data, token_value_comparator)
-
-  -- error handling for removal of token arranger
-  if self == nil then
-    for _, token in ipairs(getObjectsWithTag("tempToken")) do token.destruct() end
-    return
-  end
+  table.sort(data, tokenValueComparator)
 
   -- laying out the tokens
-  local pos           = self.getPosition() + Vector(3.55, -0.05, -3.95)
-  if percentage then
-    pos.z = self.getPosition().z - 7
-  end
-  local location      = { x = pos.x, y = pos.y, z = pos.z }
-  local current_value = data[1].value
-  local token_count = 0
-	local value_count = 1
-	local cumulative_percentage = 0
-  local current_token = false
+  local pos = self.getPosition() + Vector(3.55, -0.05, -3.95)
+  if percentage then pos.z = pos.z - 3.05 end
+
+  local location = { x = pos.x, y = pos.y, z = pos.z }
+  local currentValue = data[1].value
+  local tokenCount = 0
+	local valueCount = 1
```

Comment:
Why the indent here?

Reply:
Weird, that's not the case for me locally: 
![image](https://user-images.githubusercontent.com/97286811/230870038-9646db57-6445-45d8-8f08-399596e651a3.png)